### ISSUE TITLE
Make this a more general "Plotting for HEP" lesson

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-matplotlib#142]
This would open room for bonus lessons on visualization with other libraries, e.g., using  the `hist` library for histograms

**Comment by amangoel185** [comment]
I really like this idea, we could extend into the Scikit-HEP ecosystem!

**Comment by klieret** [comment]
Yeah, and I think it would also be useful to at least mention `seaborn` which can be very nice for statistical analysis

**Comment by amangoel185** [comment]
True! Although I wonder if `seaborn` is used in HEP widely?
